src/Dev/CsvBulkLoader.php

```diff
@@ -254,11 +254,11 @@ protected function processChunk($filepath, $preview = false)
         Deprecation::notice('4.12.0', 'Process rows individually instead');
         $results = BulkLoader_Result::create();
 
-        $csv = new CSVParser(
-            $filepath,
-            $this->delimiter,
-            $this->enclosure
-        );
+        $delimiter = $this->delimiter;
+        $enclosure = $this->enclosure;
+        $csv = Deprecation::withNoReplacement(function () use ($filepath, $delimiter, $enclosure) {
+            return new CSVParser($filepath, $delimiter, $enclosure);
+        });
```

Comment:
This is the only usage of this class in our code that I could find.

Since this whole method is deprecated, it's easier to just wrap this instantiation than to replace it with the `Reader` class. It's gone in CMS 5 anyway.